code-dot-org/code-dot-org
# Navigate to script overview on save and close of lesson edit for lessons without lesson plan

If you set has_lesson_plan to false on the lesson edit page and then hit "save and close" it will take you to the script overview page instead of the lesson plan page. 

![navigate-lesson-wo-lp](https://user-images.githubusercontent.com/208083/105908564-5b19ab80-5ff4-11eb-9a18-736407fcd5f9.gif)

You also can't directly access a lesson plan for a lesson for which has_lesson_plan is false.

![Screen Shot 2021-01-26 at 4 36 06 PM](https://user-images.githubusercontent.com/208083/105908757-97e5a280-5ff4-11eb-8079-aa32f223dbe5.png)


## Links



- [lockable and lessons without lesson plan planning](https://docs.google.com/document/d/1sAAVnwv3rXQEK-qa8JnkNXFfZ0vjWQyzJ9Yx_pG4bdg/edit#)
- [jira](https://codedotorg.atlassian.net/browse/PLAT-721)

## Testing story

- Add test to check that client side navigates correctly based on has_lesson_plan
- Add tests to check that forbidden from going to a lesson plan page for lesson without lesson plan

# Reviewer Checklist:
